### DISCUSSION TITLE
Welcome to TIC-80 Discussions!

### DISCUSSION THREAD

**Discussion by nesbox** [nesbox/TIC-80#1285] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RobLoach** [comment]
Greetings!  :wave: Glad you set this up.

**Comment by 12buntu** [comment]
Hi, I'm Kyle, a student working with TIC-80 for fun. I'd like to contribute to TIC-80 at some point soon, but I don't feel quite skilled enough to contribute much right now. I'm learning Wren right now, so perhaps I'll add some more documentation for Wren & TIC-80.

**Comment by potato25** [comment]
Hi, I'm Potato Imaginator . I'm a YouTuber i guess 🙃

**Comment by darltrash** [comment]
Hi, my name is Neil and I like making retro-like-ish games with Lua.
I also make music, level design, drawing, and etc. so, tic80 was some sort of "Perfect fit" for me!
